code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -107,12 +120,14 @@
 		if(insert(AM) == -1) // limit reached
 			break
 
-/obj/structure/closet/proc/open()
-	if(opened || !can_open())
+/obj/structure/closet/proc/open(mob/living/user)
+	if(opened || !can_open(user))
 		return
 	playsound(loc, open_sound, 15, 1, -3)
 	opened = 1
-	density = 0
+	if(!dense_when_open)
+		density = 0
+	climb_time *= 0.5 //it's faster to climb onto an open thing

```

Comment:
You're probably right if the box is taller than waist-high, but if it's lower than that it's probably easier to step into than climb onto.
I'm implying I'm considering crates shorter than waist-high, where this speed boost makes sense.